### ISSUE TITLE
dnt build is crashing with a panic

### ISSUE THREAD

**Issue by bebraw** [denoland/dnt#104]
In order to run [Gustwind](https://gustwind.js.org/) through Node, I set up dnt in my project. The problem is that dnt is crashing with a panic.

Here's how to reproduce:

1. Check out https://github.com/gustwindjs/gustwind/tree/feat/compile-with-dnt
2. `vr build:npm` (needs [Velociraptor](https://github.com/umbopepato/velociraptor))

After this you should see something like below:

```
[dnt] Transforming...
panicked at 'called `Option::unwrap()` on a `None` value', rs-lib\src\mappings.rs:134:52

Stack:

Error
    at imports.wbg.__wbg_new_693216e109162396 (https://deno.land/x/dnt@0.16.0/lib/pkg/dnt_wasm.js:313:19)
    at <anonymous> (wasm://wasm/00ff96be:1:3074224)
    at <anonymous> (wasm://wasm/00ff96be:1:3375649)
    at <anonymous> (wasm://wasm/00ff96be:1:3256299)
    at <anonymous> (wasm://wasm/00ff96be:1:3311261)
    at <anonymous> (wasm://wasm/00ff96be:1:3362231)
    at <anonymous> (wasm://wasm/00ff96be:1:3365494)
    at <anonymous> (wasm://wasm/00ff96be:1:3358903)
    at <anonymous> (wasm://wasm/00ff96be:1:690751)
    at <anonymous> (wasm://wasm/00ff96be:1:602901)


error: Uncaught (in promise) RuntimeError: unreachable
    at <anonymous> (wasm://wasm/00ff96be:1:3256353)
    at <anonymous> (wasm://wasm/00ff96be:1:3311261)
    at <anonymous> (wasm://wasm/00ff96be:1:3362231)
    at <anonymous> (wasm://wasm/00ff96be:1:3365494)
    at <anonymous> (wasm://wasm/00ff96be:1:3358903)
    at <anonymous> (wasm://wasm/00ff96be:1:690751)
    at <anonymous> (wasm://wasm/00ff96be:1:602901)
    at <anonymous> (wasm://wasm/00ff96be:1:1979352)
    at <anonymous> (wasm://wasm/00ff96be:1:3024452)
    at <anonymous> (wasm://wasm/00ff96be:1:3370059)
error: Failed at the build:npm script
```

Is it possible to get a cleaner trace somehow to understand what's going on?

---

Here's my Deno version info for reference:

```
deno 1.17.3 (release, x86_64-apple-darwin)
v8 9.7.106.15
typescript 4.5.2
```

**Comment by kt3k** [comment]
I experienced the same error with [deno_license_checker](https://github.com/kt3k/deno_license_checker).

When I tried to update dnt to v0.16.0, I saw a very similar panic log https://github.com/kt3k/deno_license_checker/runs/4877507047?check_suite_focus=true

When I downgraded dnt to v0.15.0, the error didn't happen.

**Comment by dsherret** [comment]
@bebraw sorry, I missed this yesterday! This is fixed now in 0.16.1.

One issue you're going to run into is that some of your dependencies (`deno info mod.ts`) use an old std version and it seems one of the hashing modules previously had a top level await, which is not supported in CJS modules. So dnt has the following error:

```
[dnt] Top level await cannot be used when distributing CommonJS (See deps/deno_land/std_0.97.0/hash/_wasm/hash.ts 15:1). Please re-organize your code to not use a top level await or only distribute an ESM module by setting the 'cjs' build option to false.
error: Uncaught (in promise) Error: Build failed due to top level await when creating CommonJS package.
```

It seems the newer versions of std do not have a top level await there, so upgrading those dependencies might fix that issue.

That said, after I turned of `cjs: false`, I ran into many similar issues that we're experiencing in #14 and also ran into #106. Also ran into https://github.com/denoland/node_deno_shims/issues/60
